vignettes/rticles.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Get started"
+output: rmarkdown::html_vignette
```

Comment:
I am wondering if we should add a vignette or not. Currently, there is none on [CRAN](https://cran.r-project.org/web/packages/rticles/). We could just do articles for pkgdown website for this package.

This would mean adding whole vignettes folder to Rbuildignore